docs/api-reference/extensions/clip-extension.md

```diff
@@ -0,0 +1,59 @@
+
+# ClipExtension
+
+The `ClipExtension` adds support for clipping rendered layers by rectangular bounds.
+
+```js
+import {BitmapLayer} from '@deck.gl/layers';
+import {ClipExtension} from '@deck.gl/extensions';
+
+const layer = new BitmapLayer({
```

Comment:
I recommend using a GeoJsonLayer as example. The code here does not illustrate why clipping is useful.